serde_derive/src/internals/attr.rs

```diff
@@ -192,6 +238,13 @@ impl Name {
     }
 }
 
+pub type VariantNames = Names<VariantName>;
+pub type Name = Names<String>;
```

Comment:
Is there a more harmonious pair of names to use here? What is `Name` used for &mdash; field names? If so, `FieldNames`?